awscli/testutils.py

```diff
@@ -181,6 +181,46 @@ def create_bucket(session, name=None, region=None):
     return bucket_name
 
 
+def create_dir_bucket(session, name=None, region=None, az=None):
+    """
+    Creates a S3 directory bucket
+    :returns: the name of the bucket created
+    """
+    if not region:
+        region = 'us-west-2'
+    client = session.create_client('s3', region_name=region)
+    if not az:
+        az = 'usw2-az1'
```

Comment:
We might consider making the `region` and `az` input a tuple since they're inherently bound. i.e. if I pass `us-east-1` but forget to add the `az`, I don't think we want those disjoint, do we?

Something like this:
```suggestion
    if not location:
        location = ('us-west-2', 'usw2-az1')
    region, az = location
    client = session.create_client('s3', region_name=region)
```